python/graphstorm/dataloading/dataset.py

```diff
@@ -154,6 +154,45 @@ def prepare_batch_edge_input(g, input_edges,
                 feat[etype] = th.cat(feats, dim=1)
     return feat
 
+def prepare_blocks_edge_feats(g, input_blocks, efeat_fields, device='cpu'):
```

Comment:
Is this change related to this PR?

Reply:
Yes, the new `GSGnnNodePredictionRealtimeInferrer` class' `infer()` function need it to get features.